### DISCUSSION TITLE
電磁ブレーキコマンドの挙動

### DISCUSSION THREAD

**Discussion by bbm-soka** [ponoor/step-series-support#35] [General]
Step400デバイスに電磁ブレーキ付きモーターを使用しています。
microSDカードより、"Electromagnetic brake enable"をTrueに設定しているのですが、
Step400が起動してからブレーキをactivateすると"Ignore Command"と表示され、activateできないようです。

oscにて再度、"Electromagnetic brake enable"をTrueでコマンド送信すると、activateできるようになります。

何か考えられる原因はありますでしょうか？

**Comment by kanta** [comment]
レポートありがとうございます。検証してみたところ、同じ症状が再現しました。
これはソースコードにバグがあり、configファイルの内容にかかわらず、起動時は電磁ブレーキモードが無効になる状態でした。修正したものをv1.3.1としてアップデートしましたので、お手数をおかけしますが以下よりダウンロードいただき、新しいファームウェアを書き込んでみていただけないでしょうか。
https://github.com/ponoor/step-series-universal-firmware/releases/tag/v1.3.1

書き込み方法はこちらをご参照ください。
https://ponoor.com/docs/step-series/technical-information/compile-the-arduino-sketch/
